Sources/Components/Button/Playground/ButtonPlayground.swift

```diff
@@ -95,12 +95,10 @@ public class ButtonPlayground: UIView {
             destructiveButton.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -.largeSpacing),
 
             flatButton.topAnchor.constraint(equalTo: destructiveButton.bottomAnchor, constant: .mediumLargeSpacing),
-            flatButton.leadingAnchor.constraint(equalTo: leadingAnchor, constant: .largeSpacing),
-            flatButton.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -.largeSpacing),
+            flatButton.centerXAnchor.constraint(equalTo: centerXAnchor),
```

Comment:
This will cause the button to bleed outside the edge of the screen if the button title is too large. I think the `trailingAnchor` and `leadingAnchor` are important constraints.

Reply:
Event though it will most likely never be that large I think it's good practice.

Reply:
Good point. Might be good to have an example that shows that in case we don’t have any.

Reply:
That's a good point, but what I saw was that the button was bigger than what it actually looked like. You could select it even when you pressed outside of the button. 
Should I add those constraints back? @ulrikus

Reply:
Maybe change the "_equalTo_" to "_lessThanOrEqualTo_" to that it snaps to the size of the button. I think that will do it. You may need to have the centerXAnchor also, I'm not sure.

Reply:
Now: <img width="438" alt="screen shot 2018-01-31 at 09 56 07" src="https://user-images.githubusercontent.com/1401033/35613674-12207762-066d-11e8-9bd8-3967e53df8bd.png">

Before: 
<img width="436" alt="screen shot 2018-01-31 at 09 57 22" src="https://user-images.githubusercontent.com/1401033/35613724-2b13b73e-066d-11e8-9ef3-84941f8927b0.png">

Reply:
```
            flatButton.topAnchor.constraint(equalTo: destructiveButton.bottomAnchor, constant: .mediumLargeSpacing),
            flatButton.leadingAnchor.constraint(lessThanOrEqualTo: leadingAnchor, constant: .largeSpacing),
            flatButton.trailingAnchor.constraint(lessThanOrEqualTo: trailingAnchor),
            flatButton.centerXAnchor.constraint(equalTo: centerXAnchor),
```
<img width="430" alt="screen shot 2018-01-31 at 10 00 24" src="https://user-images.githubusercontent.com/1401033/35613914-a2d94612-066d-11e8-81c4-cece35204888.png">

Reply:
It makes sense in flat buttons has a background color.